Packages/ClientRuntime/Sources/PrimitiveTypeExtensions/Number+Extension.swift

```diff
@@ -28,4 +28,3 @@ public extension Double {
         }
     }
 }
-
```

Comment:
revert?

Reply:
hm weird, didnt touch this file.  must have been the linter